src/odemis/driver/orsay.py

```diff
@@ -2760,3 +2771,242 @@ def terminate(self):
             self.stop()
             self._executor.shutdown()
             self._executor = None
+
+
+NMBR_APERTURES = 30
+
+
+class FIBAperture(model.Actuator):
+    def __init__(self, name, role, parent, **kwargs):
+        # TODO K.K. define a more sane range for the ranges
+        axes_def = {"x": model.Axis(unit="m", range=(-6e-3, 6e-3)),
+                    "y": model.Axis(unit="m", range=(-6e-3, 6e-3))}
```

Comment:
Can you get the ranges from the .Max and .Min of the orsay parameters maybe? Otherwise, put them in a constant.